packages/playground/wordpress/src/version-detect.ts

```diff
@@ -19,13 +19,19 @@ export async function getLoadedWordPressVersion(
 }
 
 /**
- * Returns a WordPress build version string known to Playground
+ * Returns a WordPress build version string supported by Playground,
  * for a given WordPress version string.
  *
+ * Playground supports the last 4 major.minor versions of WordPress,
```

Comment:
This is incorrect, though. There's nothing in this function that limits the returned version to the last 4 major releases.

Reply:
True, I wanted to explain what supported means with that sentence, but it's actually not relevant.

Reply:
I removed the mention of support in [0d062ea](https://github.com/WordPress/wordpress-playground/pull/1987/commits/0d062ea3700af84a02cac329ff09378ced78164d) because it's not relevant.